reproductions/HttpMessageHandler.StackOverflowException/HttpMessageHandler.StackOverflowException.csproj

```diff
@@ -2,7 +2,6 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp2.0</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
```

Comment:
Can we remove this line, too? I think it's defined in the shared props file.

Reply:
Cleaned up in 60d0681caa1f6a10aaccab537806c714ead22add